tf/modules/coredns/main.tf

```
@@ -17,7 +17,7 @@ resource "kubernetes_config_map" "original_coredns" {
 }
 
 locals {
-  rewrite_line      = "rewrite name ${var.harbor_host} ${var.nginx_service_url}"
+  rewrite_line      = indent(4, "rewrite name ${var.harbor_host} ${var.nginx_service_url}\nrewrite name ${var.sonarqube_host} ${var.nginx_service_url}")
```

Probably overkill for two hosts, but if we add more:
```suggestion
  rewrite_line      = join("\n", formatlist("rewrite %s ${var.nginx_service_url}", [${var.harbor_host}, ${var.sonarqube_host}]))
  
  

```

---

this works, but we still need the indentation I think